index.js

```diff
@@ -22,10 +22,12 @@ module.exports = {
     'eol-last':                    ['error'],
     'eqeqeq':                      ['error'],
     'indent':                      ['error', 2, {SwitchCase: 1}],
+    'key-spacing':                 ['error', {beforeColon: false, afterColon: true}],
```

Comment:
I guess the alignment makes sense. I see code like this in a number places. Breaking that wouldn't be so great.

```
      routes: {
        'report/:reportId':             this.routeHandlers.load,
        'report/:reportId/:reportName': this.routeHandlers.load,
        'reset':                        this.routeHandlers.reset,
        'learn':                        this.routeHandlers.learn,
        ':jsurl':                       this.routeHandlers.jsurl,
        '':                             this.routeHandlers.index,
      },
```